### ISSUE TITLE
A question if Garden of Glass features and recipes can be used in non-GoG worldtype?

### ISSUE THREAD

**Issue by FloratheFernFern** [Vazkii/Botania#2752]
Forge version 1.12.2-14.23.4.2705
Botania r1.10-355.jar
Minecraft 1.12.2

Hello!
**I am trying out the Garden of Glass add on and I am wondering if it is possible to use those features in other world types like customized and super-flat, not just Garden of Glass generation**. I don't necessarily want to be stuck on a sky block island since other mods I am using use world generation for their structures and ores but still want that isolated feeling of a skyblock and something to look forward to in building away from the island.

**Comment by williewillus** [comment]
Yes, the configuration and recipe changes apply any time the GoG mod is installed, regardless of the world type

**Comment by FloratheFernFern** [comment]
I'm sorry I wasn't clear in my original issue post/question, but I mean to say that I have already tried using the recipes and features in the normal world, but so far are only able to use the recipes. I also want to be able to use the pebble, waterbowl, and buffed orechid of GoG, but seeing that I can't make pebbles or waterbowls in normal,customized or superflat worlds, I don't think rebuffed orechids would work and that GoG features aren't, for me, working in the default type worlds. I've tried splicing in customized generation into a GoG generated skyblock world by using NBTexplorer, but then for some reason I can't make any more pebbles or water bowls. These are the features I really want since they are important for a sort of survival in the "dire conditions" type of world I have created in customized worlds.
